pkg/extract/types/attack/attack.go

```diff
@@ -3,3 +3,7 @@ package attack
 type Attack struct{}
 
 func (d *Attack) Sort() {}
+
+func Compare(x, y Attack) int {
+	return 0
```

Comment:
Are these "return 0" implementaion NOT-YET-IMPLEMENTED? Or actually ZERO?

Reply:
not yet implemented